scrapy/utils/project.py

```diff
@@ -71,3 +71,16 @@ def get_project_settings():
         settings.setdict(env_overrides, priority='project')
 
     return settings
+
+def get_project_path():
+    """Return the Python path of the current project"""
+    # XXX: This is far from optimal. It fails when the settings module lives
+    #      outside of the project package, and when the project package lives
+    #      within another package.
+    if not inside_project():
+        raise NotConfigured("Not inside a project")
+    settings_module_path = os.environ.get(ENVVAR)
+    if not settings_module_path:
+        raise NotConfigured("Unable to locate project's python path")
+    return settings_module_path.split('.')[0]
```

Comment:
Another thing we could also do is to force users to define the full path of the addons inside their projects, like when we define components paths in the settings, I think I like this option better.